### DISCUSSION TITLE
optional round first and last children in join group

### DISCUSSION THREAD

**Discussion by meritozh** [saadeghi/daisyui#2418] [Ideas / feature requests]
as #2408 say, first and last children have unexpect rounded border (for me), how about make this behavior optional?

link: https://play.tailwindcss.com/zemYLtPZQl

![image](https://github.com/saadeghi/daisyui/assets/7543300/a9a34c11-fe97-4dcf-99bf-8e1b599699dd)

**Comment by meritozh** [comment]
@saadeghi Would you like this idea? I can make a pr for this.

**Reply by saadeghi** [reply]
Thank you.  
Answered here: https://github.com/saadeghi/daisyui/issues/2408
